content/contexts/recipeDetails.js

```diff
@@ -26,7 +27,8 @@ function reducer(state, action) {
     case ACTION_UPDATE_DATA:
       return {
         ...state,
-        data: action.data,
+        data: action.data || state.data,
+        statusData: action.statusData || state.statusData,
```

Comment:
I suppose `??` would make sense here. It could be undefined/null if the action passed to dispatch contained only one of these.

Reply:
I looked around at where `ACTION_UPDATE_DATA` is used, and I see that there are a lot of times that we would only have one of the fields, especially because the editing forms seem to share this context. Alright.